src/bin_editor/Oni2_editor.re

```diff
@@ -74,6 +76,70 @@ let cliOptions =
       },
     ~printVersion,
   );
+
+let initWorkingDirectory = () => {
+  let path =
+    switch (cliOptions.folder) {
+    | Some(folder) => folder
+    | None =>
+      switch (Store.Persistence.(Store.get(Global.workspace, Global.store))) {
+      | Some(path) => path
+      | None =>
+        switch (Revery.Environment.getWorkingDirectory()) {
+        | "/" => Sys.getenv_opt("HOME") |> Option.value(~default="/")
```

Comment:
We may want to use this method: https://github.com/onivim/oni2/blob/5d4c99a9409429358e171abf7c9e67b16bfb1bd2/src/Core/Filesystem.re#L216

Reply:
Good idea!

Reply:
Used `Dir.User.document` and `Dir.home` instead, as discussed above.